srcpkgs/botan/update

```diff
@@ -1 +1,2 @@
 pattern='\K[0-9]+\.[0-9]*[02468]\.[0-9]+'
+ignore="*3.0.*" # alpha release series
```

Comment:
Will the non alpha release be 3.1.x?

Reply:
I just ran the update check on botan again with the `update` file deleted and it seems like the pattern was broken. It is working properly now with no need to have the ignore.